qiita_pet/templates/list_studies.html

```diff
@@ -38,7 +44,7 @@
   moi.send('sel', proc_data);
 }
 
-function show_alert(data) { bootstrapAlert(data + ' samples selected.', "success", 2000); }
+function show_alert(data) { bootstrapAlert(data + ' samples selected.', "success", 2000); $('#dflt-sel-info').css('color', 'rgb(0, 160, 0)');}
```

Comment:
A byte? There are likely other larger areas of connection overhead that
would be more effective to worry about if this is a problem. We can do .min
versions later which would resolve extraneous whitespace, likely better to
worry about readability right now
On Apr 23, 2015 10:03 PM, "Joshua Shorenstein" <notifications@github.com>
wrote:

> In qiita_pet/templates/list_studies.html
> <https://github.com/biocore/qiita/pull/1101#discussion_r29022483>:
>
> > @@ -38,7 +44,7 @@
> >    moi.send('sel', proc_data);
> >  }
> >
> > -function show_alert(data) { bootstrapAlert(data + ' samples selected.', "success", 2000); }
> > +function show_alert(data) { bootstrapAlert(data + ' samples selected.', "success", 2000); $('#dflt-sel-info').css('color', 'rgb(0, 160, 0)');}
>
> I don't, but javascript doesn't care about line breaks and it saves some
> transmit info.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/qiita/pull/1101/files#r29022483>.
>